actions/pacstrap_action.go

```diff
@@ -0,0 +1,155 @@
+/*
+Pacstrap Action
+
+Construct the target rootfs with pacstrap tool.
+
+Yaml syntax:
+ - action: pacstrap
+   mirror: <url with placeholders>
+   repositories: <list of repositories>
+
+Mandatory properties:
+
+ - mirror -- the full url for the repository, with placeholders for
+   $arch and $repo as needed, as would be found in mirrorlist
+
+Optional properties:
+ - repositories -- list of repositories to use for packages selection.
+   Properties for repositories are described below.
+
```

Comment:
I'm hoping this new config data works better in terms of handling:
- [Avoiding the mirror layout patch](https://github.com/santoshmahto7/debos/commit/fe2c4bfe0b78713baba85ef373e5c6d0774a155b#commitcomment-53537458)
- [Keeping the siglevel patch (or at least functionality)](https://github.com/santoshmahto7/debos/commit/81052fb5011417d31ee0426f1d6b63d9316ee8ea#commitcomment-53538999)
- [Dropping the user defined repositories patch](https://github.com/santoshmahto7/debos/commit/ba40558e45f406302b08f663dbd956cb8a12bac1#commitcomment-53537706); although you can still select your repositories, the default is correct, and there's no possibility of mixing and matching between mirrors. I left the option to select them in because even arch has more repositories than this at need, and I couldn't see an obvious cost to doing so.

Reply:
Fully agree - the elaborate mirror-layout is gone. Having default repositories with override sounds reasonable - say you want a normal x86-64, or you want to add multilib on top. Or even switch to the testing version of said repos.